.github/workflows/gradle-build-feature.yml

```diff
@@ -0,0 +1,19 @@
+name: Gradle Build Feature Branch
+on:
+  push:
+    branches: [develop]
+    paths:
+      - 'feature/*'
```

Comment:
I'm confused by this...I think it belongs in branches, no?  This would trigger only if a file was modified in a directory called "feature".  I'm guessing that was not the intention.

https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestpaths

Reply:
I was confused also. Thank you for the link. I have updated the code to read:
```
on:
  push:
    branches:
      - 'feature/**'
      - 'bug/**'
```